tests/fsharp/tools/eval/test.fsx

```diff
@@ -1477,6 +1477,269 @@ module EvaluationTests =
 
         Eval <@ Array.average [| 0.0 .. 1.0 .. 10000.0 |] @> 
 
+    module GenericInlinedOperationsStillDynamicallyAvailableTests = 
+
+        let inline test1 (x: 'T) = 
+            checkEval ("vroievr093-(" + typeof<'T>.ToString() + ")") (<@ LanguagePrimitives.GenericZero<_> @>)  x
+
+        test1 0s
+        test1 0
+        test1 0L
+        test1 0n
+        test1 0uy
+        test1 0us 
+        test1 0u
+        test1 0UL
+        test1 0un 
+        test1 0.0
+        test1 0.0f
+        test1 0M
+
+        let inline test2 (x: 'T) = 
+           checkEval ("vroievr095f3-(" + typeof<'T>.ToString() + ")") (<@ LanguagePrimitives.GenericOne<_> @>)  x
+
+        test2 1s
+        test2 1
+        test2 1L
+        test2 1n
+        test2 1uy
+        test2 1us 
+        test2 1u
+        test2 1UL
+        test2 1un 
+        test2 1.0
+        test2 1.0f
+        test2 1M
+
+        let inline test3 (x: 'T) y = 
+           checkEval ("vroievr096cfqw-(" + typeof<'T>.ToString() + ")") (<@ x + y @>)  (x + y)
+
+        test3 3s 4s
+        test3 3L 4L
+
+        let inline test4 (x: 'T) =
+           checkEval ("vroievrgerwwge41-(" + typeof<'T>.ToString() + ")") (<@ abs x @>)  (abs x)
+           checkEval ("vroievrgerwwge42-(" + typeof<'T>.ToString() + ")") (<@ sin x @>)  (sin x)
+           checkEval ("vroievrgerwwge43-(" + typeof<'T>.ToString() + ")") (<@ cos x @>)  (cos x)
+           checkEval ("vroievrgerwwge44-(" + typeof<'T>.ToString() + ")") (<@ tan x @>)  (tan x)
+           checkEval ("vroievrgerwwge45-(" + typeof<'T>.ToString() + ")") (<@ sinh x @>)  (sinh x)
+           checkEval ("vroievrgerwwge46-(" + typeof<'T>.ToString() + ")") (<@ cosh x @>)  (cosh x)
+           checkEval ("vroievrgerwwge47-(" + typeof<'T>.ToString() + ")") (<@ ceil x @>)  (ceil x)
+           checkEval ("vroievrgerwwge49-(" + typeof<'T>.ToString() + ")") (<@ tanh x @>)  (tanh x)
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ floor x @>)  (floor x)
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ truncate x @>)  (truncate x)
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ round x @>)  (round x)
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ log (abs x) @>)  (log (abs x))
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ sqrt (abs x) @>)  (sqrt (abs x))
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ log10 (abs x) @>)  (log10 (abs x))
+           checkEval ("vroievrgerwwge48-(" + typeof<'T>.ToString() + ")") (<@ exp (abs x) @>)  (exp (abs x))
+
+        test4 -1.1
+        test4 -1.1f
+
+        let inline test5 (x: 'T) y = 
+           checkEval ("vrbrer096cfqw-(" + typeof<'T>.ToString() + ")") (<@ (fun x2 y2 -> x2 + y2) x y @>) (x + y)
+
+        test5 3s 4s
+        test5 3L 4L
+(*
```

Comment:
Was this meant to be commented out?

Reply:
No,thanks - I've uncommented it (will see in CI if it causes problems)